MaxiOps/gasStation/2024-04/lm/withdraw_gas_lz_poker_mainnet.json

```diff
@@ -0,0 +1,34 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1714062062268,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.16.5",
+    "createdFromSafeAddress": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+    "createdFromOwnerAddress": "",
+    "checksum": "0xf0714692670b0a3bd04892d7ab013338695df1b2cabd731efa85989dce636228"
+  },
+  "transactions": [
```

Comment:
We need to send the ETH back to the gas station as well.  Can you include a second tx in the list send to the resulting eth to the gas station such that there is no end balance change in the safe?

Here's an example: https://github.com/BalancerMaxis/multisig-ops/blob/e35ec951a1942bfe835adc0dd473390f458a8dcc/BIPs/2024-W11/BIP-556.json#L14